src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFacts.cs

```diff
@@ -519,6 +519,14 @@ internal partial interface ISyntaxFacts
         SyntaxNode GetExpressionOfThrowExpression(SyntaxNode throwExpression);
         bool IsThrowStatement(SyntaxNode node);
         bool IsLocalFunction(SyntaxNode node);
+
+        /// <summary>
+        /// Returns the topmost ancestor node for the IOperation tree structure that contains
+        /// the node passed in. 
```

Comment:
doc the invariants around CompilationUnits though.

Reply:
Removed in latest iteration